Frends.FTP.UploadFiles/Frends.FTP.UploadFiles.Tests/Lib/UploadFilesTestBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+using NUnit.Framework;
+
+namespace Frends.FTP.UploadFiles.Tests;
+
+/// <summary>
+/// Provides simple dir and helper methods for local file creation
```

Comment:
IMO here or somewhere should be documented what needs to be done to run these tests. eg. cd -> tests dir and run docker-compose ...

Reply:
Added to README